packages/lesswrong/components/editor/CKPostEditor.tsx

```diff
@@ -595,6 +595,7 @@ const CKPostEditor = ({
         dialogues: dialogueConfiguration
       }}
     />}
+    {post.collabEditorDialogue ? <DialogueEditorFeedback post={post} /> : null}
```

Comment:
Unfortunate that this will persist in being displayed even after the user clicks on it once